.github/workflows/automated-version-update-pr.yml

```diff
@@ -15,8 +15,8 @@ jobs:
     runs-on: ubuntu-latest
     env:
       GUTENBERG_MOBILE_VERSION: ${{ github.event.inputs.gutenbergMobileVersion }}
-      PR_TITLE: ${{ github.event.inputs.title || 'Automated gutenberg-mobile version update for ' +  ${{ github.event.inputs.gutenbergMobileVersion }}   }}
-      PR_BODY: ${{ github.event.inputs.body || 'This PR incorporates changes from [gutenberg-mobile repo](https://github.com/wordpress-mobile/gutenberg-mobile).' }} 
+      PR_TITLE: ${{ github.event.inputs.title || format('{{Automated gutenberg-mobile version update for {0} !}}', github.event.inputs.gutenbergMobileVersion ) }}
+      PR_BODY: ${{ github.event.inputs.body || format( '{{This PR incorporates changes from {0}.}}', github.event.inputs.prURL ) }}'' 
```

Comment:
Are the trailing `''` there in error?

Reply:
Yes, these changes shouldn't be there. Good catch!